coconut/configuration/configuration.go

```diff
@@ -53,12 +54,15 @@ type ConfigurationCall func(*configuration.ConsulSource, *cobra.Command, []strin
 
 var componentsPath = "o2/components/"
 
-var InputRegex, _ = regexp.Compile(`^([a-zA-Z0-9-]+)(\/[a-z-A-Z0-9-]+)?(\@[0-9]+)?$$`)
+var InputRegex, __ = regexp.Compile(`^([a-zA-Z0-9-]+)(\/[a-z-A-Z0-9-]+){1}(\@[0-9]+)?$`)
```

Comment:
`regex.MustCompile` instead of ignoring error please.

Reply:
Argh, bad reverting. Updated now